app/index.html

```diff
@@ -128,6 +128,21 @@ <h2 class="title-margin-bottom">News &#x26; Events</h2>
                
                <div class="eight columns">
                    
+                   <h4>Notice of Proposed Changes to CFA Bylaws</h4>
+                    <p>December 16, 2019</p>

```

Comment:
Nit: it would be nice to fix the indentation on this. Currently all these paragraphs are one space in past the `h4`.